py3status/modules/xrandr_rotate.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+
+"""
+Switch between horizontal and vertical screen rotation on a single click.
+
+Configuration parameters:
+    - cache_timeout: how often to refresh this module.
```

Comment:
being picky,

can you remove the `- ` from these parameters, and if you are really nice put the default in parenthesis

like this

```
    format: a string that formats the output, can include placeholders.
        (default is '{icon}')
```

This helps with creating the README.md

I need to document this somewhere

this is also the case with the placeholders

eg

```
    {icon} a rotation icon, ...
```